pyproject.toml

```
@@ -7,12 +7,12 @@ requires-python = ">=3.8"
 authors = [
     {name = "Inada Naoki", email = "songofacandy@gmail.com"}
 ]
-license = {text = "GNU General Public License v2 (GPLv2)"}
+license = {text = "GNU General Public License v2 or later (GPLv2+)"}
```

Use the SPDX identifier (https://spdx.org/licenses/) here please:

```suggestion
license = {text = "GPL-2.0-or-later"}
```

---

Why?

https://packaging.python.org/en/latest/guides/writing-pyproject-toml/#license

No recommendation for using SPDX identifier here.

---

> If you are using a standard, well-known license, it is not necessary to use this field. Instead, you should use one of the [classifiers](https://packaging.python.org/en/latest/guides/writing-pyproject-toml/#classifiers) starting with License ::. (As a general rule, it is a good idea to use a standard, well-known license, both to avoid confusion and because some organizations avoid software whose license is unapproved.)
>
> https://packaging.python.org/en/latest/guides/writing-pyproject-toml/#license

I can remove this field, because I use License classifier later.

---

> A practical string value for the license key has been purposefully left out to allow for a future PEP to specify support for [SPDX](https://spdx.dev/) expressions (the same logic applies to any sort of “type” field specifying what license the file or text represents).
>
> https://peps.python.org/pep-0621/#license

OK. It is planned to have some PEP for writing SPDX expressions here. But there is no PEP for now.

---

Only if the `license` field is specified does the PyPI API return the `license` key in its response currently unfortunately. This helps license scanners check for licenses of dependencies. Could we keep it with the text you initially proposed for now?

Either way, [PEP 639](https://peps.python.org/pep-0639/) has been provisionally accepted recently which will improve on this situation. If you prefer we can also wait for support for this PEP to be added.

---

As PEP 639, `license = {...}` is deprecated and `license = "expression"` is introduced. So I won't `license = {text="expression"}`.

I don't delete deprecated `license = {text="name"}` until PEP 639 is implemented at PyPI and packaging tools.

---

Thanks, sounds good!